command_handler.py

```diff
@@ -15,18 +14,19 @@ async def handle(self, message: types.Message):
             logger.debug("USER TYPED UNKNOWN COMMAND " + message.text)
 
     async def __handleHelp(self, message):
-        await message.reply(text="This bot is going to manage cpop.tw.\n"
+        await message.reply(text="This bot is going to manage cpop\.tw\.\n"
```

Comment:
is there really a need to escape a **dot**? I think dot should work with no escaping

Reply:
yes, need to escape dots here because of MarkdownV2
> In all other places characters '_', '*', '[', ']', '(', ')', '~', '`', '>', '#', '+', '-', '=', '|', '{', '}', '.', '!' must be escaped with the preceding character '\'.
https://core.telegram.org/bots/api#formatting-options

If those characters are not escaped, telegram server will reply with a warning about the character is not escaped and fail to post the message.